.ci/Jenkinsfile

```diff
@@ -311,7 +311,8 @@ def generateFunctionalTestStep(Map args = [:]){
   def workerLabels = "${platform} && immutable"
   if (platform == "arm64") {
     goArch = "arm64"
-    workerLabels = "aarch64"
```

Comment:
Temporarily reverting this, as I'm observing provisioning times for worker are slower, reaching build timeouts more frequently